lforms-converter.js

```diff
@@ -398,6 +398,9 @@ function doSkipLogic(root) {
 
   traverseItems(root, function(item, ancestors) {
     if(item.skipLogic) {
+      if (item.skipLogic.condition != undefined && item.skipLogic.condition.trim() === "") {
+        delete item.skipLogic;
+      } else {
       // This is target item. Parse 'condition' to look for source item
       var tokens = item.skipLogic.condition.split('=');
```

Comment:
Hi Peter,
This does not seem to be based on latest master branch. Would you please merge the changes in current master branch into this pull request?

BTW, we prefer indent size to be 2.

Thanks,
-Ajay